src/FunctionApps/DSTP/.gitignore

```diff
@@ -0,0 +1,36 @@
+# Build output
+target/
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+# IDE
+.idea/
+*.iml
+
+# macOS
+.DS_Store
+
+# Azure Functions
+local.settings.json
```

Comment:
we should definitely find some way to add this to a top level .gitignore -- this is too sensitive a file to rely on us to remember to gitignore it for each individual function app